inc/class_captcha.php

```diff
@@ -104,11 +104,7 @@ function __construct($build = false, $template = "")
 		{
 			$this->captcha_template = $template;
 
-			if($this->type == 2)
-			{
-				$this->captcha_template .= "_recaptcha";
-			}
-			elseif($this->type == 4){
+			if($this->type == 4){
```

Comment:
Code style - can you move the `{` to a new line?

Reply:
Done.